docs-src/python/choose-dev-cluster.md

```diff
@@ -0,0 +1,133 @@
+---
+id: choose-dev-cluster
+title: Choose a development Cluster
+sidebar_label: Choose dev Cluster
+description: Discover which development Cluster you should choose
+tags:
+  - developer guide
+  - python sdk
+---
+
+**Which development Cluster should you choose?**
+
+We recommend choosing a development environment based on your requirements.
+
+The source code for the Temporal Server (the orchestrating component of the Temporal Cluster) is licensed under the MIT open source license. So, in theory, anyone can take the Temporal Server code and run their Temporal Platform in any number of creative ways.
+
+However, for most developers we recommend starting by choosing one of the following:
+
+- [Local development server](#local-dev-server)
+- [Temporal Cloud](#temporal-cloud)
+- [Self-hosted Temporal Cluster](#self-hosted-temporal-cluster)
+
+:::info Temporal does not directly run your code
+
+Keep in mind that in every scenario, the “Temporal Platform” does not host and run your Workers (application code).
+It is up to you, the developer, to host your application code.
+The Temporal Platform ensures that properly written code durably executes in the face of platform-level failures.
+
+:::
+
+### Local dev server
+
+**When to use a local development server?**
+
+We recommend using the local development server if you are new to Temporal, or want to start something from scratch and don’t have a self-hosted environment ready or want to pay for a Temporal Cloud account.
+
+The Temporal CLI comes bundled with a development server and provides a fast way to start running Temporal Applications.
+
+However, the local development server does not emit any metrics.
+If you are eager to to set up Cluster-level metrics for performance tuning, we recommend using a self-hosted Cluster or Temporal Cloud.
+
+#### Start the dev server
+
+**How to start a local development server**
+
+If you have successfully installed the Temporal CLI, open a new terminal and run the following command:
+
+```bash
+temporal server start-dev
+```
+
+This command automatically starts the Temporal Web UI, creates a default Namespace, and creates an in-memory database.
+
+The Temporal Web UI serves to [http://localhost:8233](http://localhost:8233/).
+
+For more command details and options, see the [CLI reference](/cli/server/start-dev)
+
+#### Create a custom Namespace
```

Comment:
Not sure it's a common use case to create a custom namespace on the local dev server. We offer `--namespace` to change the created default. It's much more common to create a namespace on, say, the self-hosted server.

Reply:
Great feedback.